### ISSUE TITLE
Unable to create follower DBs

### ISSUE THREAD

**Issue by idan** [heroku/heroku-addon-attachments#5]
With the plugin installed, attempting to create a follower fails as following:

```
$ heroku addons:create heroku-postgresql:standard-2 --follow DATABASE_URL
 !    The database you are attempting to follow was not found.
```

After uninstalling, I'm able to create the follower without complaint.

**Comment by bjeanes** [comment]
that's odd... the `--follow XYZ` would just be passed blindly to the provider as `{"follow": "XYZ"}`. In any case, in the attachable word, things won't be identified by their config var but by their name (which in this case is just `DATABASE`).

**Comment by bjeanes** [comment]
@geemus any idea why this might be?

**Comment by idan** [comment]
@bjeanes maybe it's because I invoked with the `DATABASE_URL` identifier (vs `DATABASE`)? But regardless, `DATABASE_URL` works without the plugin, so I'd expect it to work when the plugin is installed.

**Comment by geemus** [comment]
By default, voodoo-magic (follow is translated to something else client side). I suspect we overlooked this in the plugin, see: https://github.com/heroku/heroku/blob/master/lib/heroku/command/addons.rb#L276

**Comment by chadbailey59** [comment]
I also experienced this when trying to rollback a database:

```
!! [lookbooker] (209930) ~/Code/support ▸ hsa addons:create heroku-postgresql:standard-0 --rollback cobalt --to '201
5-01-12 06:54:40+0' --admin-provisioning-secret 18anVvRiuHyFyB7KJTyyz84XDzB7ViqCERcY7waMp3A=
 !    The database you are attempting to fork was not found.
```

Same command works fine with `addons:add` sans plugin.

**Comment by chadbailey59** [comment]
I also experienced this when trying to rollback a database:

```
!! [lookbooker] (209930) ~/Code/support ▸ hsa addons:create heroku-postgresql:standard-0 --rollback cobalt --to '201
5-01-12 06:54:40+0' --admin-provisioning-secret [redacted]
 !    The database you are attempting to fork was not found.
```

The same command works fine with `addons:add` without the plugin.

**Comment by idan** [comment]
This still doesn't work even when invoking with Haiku names or other variants :cry: 

```
cosd-production · (master) ⟩ h addons                                   ~/D/h/cosd-production 1
=== cosd-production Add-on Resources
heroku-postgresql:standard-0  @singing-peacefully-5162  DATABASE, HEROKU_POSTGRESQL_PURPLE


cosd-production · (master) ⟩
heroku addons:create heroku-postgresql:standard-0 --follow @singing-peacefully-5162
 !    The database you are attempting to follow was not found.

cosd-production · (master) ⟩
heroku addons:create heroku-postgresql:standard-0 --follow singing-peacefully-5162
 !    The database you are attempting to follow was not found.

cosd-production · (master) ⟩
heroku addons:create heroku-postgresql:standard-0 --follow DATABASE
 !    The database you are attempting to follow was not found.

cosd-production · (master) ⟩
heroku addons:create heroku-postgresql:standard-0 --follow DATABASE_URL
 !    The database you are attempting to follow was not found.

cosd-production · (master) ⟩
heroku addons:create heroku-postgresql:standard-0 --follow HEROKU_POSTGRESQL_PURPLE
 !    The database you are attempting to follow was not found.

cosd-production · (master) ⟩
heroku addons:create heroku-postgresql:standard-0 --follow HEROKU_POSTGRESQL_PURPLE_URL
 !    The database you are attempting to follow was not found.

cosd-production · (master) ⟩ heroku addons:create heroku-postgresql:standard-0 --follow PURPLE
 !    The database you are attempting to follow was not found.
```

**Comment by bjeanes** [comment]
@idan indeed. It's because this hasn't been fixed yet. Likely, all of the things you just attempted should have the desired effect once fixed.

**Comment by idan** [comment]
@bjeanes :+1: Thanks!

**Comment by bjeanes** [comment]
Fixed by heroku/api#3757! This resolution is now done API-side.
